### ISSUE TITLE
permissions problem

### ISSUE THREAD

**Issue by ncoutenet** [Philipp15b/CompassEx#22]
I'm having troubles letting anyone that is not opped use this plugin.  If they are not opped looks like all they can use it the help, reset, player, info, save and remove commands.  I am using essentials group manager for permissions  
groups but I don't think it don't work.  

Moreover when I'm op and I want to privatize an ID it don't work but it tell me it's ok.

**Comment by Philipp15b** [comment]
Thanks for reporting. Could you make a list of commands that you tested and are not working?

**Comment by ncoutenet** [comment]
Yes, I can. This is the list of commands which don't work anymore:
here
direction (north/south/west/east)
bed
distance
height
hide
deathpoint
pos (X Y Z)
load (public and private)
list (public and private)
nearest (public and private)
privatize (don't work in op mode too)
remove (public and private)

**Comment by arnefm** [comment]
Exactly the same problem here.

**Comment by arnefm** [comment]
Found a workaround for the problem after looking through the code. I have only tested the features that I allow on my server, but the other commands can be enabled in the same way. See GeneralComponent.java.

In GroupManager these exact permissions are working:

```
- deathpoint     # Gives permissions for /compass deathpoint
- here              # Gives permissions for /compass here
- pos               # Gives permissions for /compass X Y Z
- dir                 # Gives permissions for /compass <north/south/east/west>
```

Unrelated: Why do I have to specify Y-coordinates when setting a compass position? The compass only functions on two dimensions, I can not see the use for vertical coordinates. You can safely assume Y==64 every time.

The help-page-num-commands setting is broken.

**Comment by Philipp15b** [comment]
Thanks for investigating. It turns out there were some old and wrong method calls in there that checked the wrong permissions. That's what I already had found out before you wrote this. Unfortunately, as you can probably see in the commit, that should only fix the `bed`, `position` and `here` commands. I don't really know what's causing the other problems.
